website/docs/r/vapp.html.markdown

```diff
@@ -122,26 +123,58 @@ resource "vcd_vapp" "web" {
+* `template_name` - (Optional; **Deprecated**) The name of the vApp Template to use
+* `memory` - (Optional; **Deprecated**) The amount of RAM (in MB) to allocate to the vApp
+* `cpus` - (Optional; **Deprecated**) The number of virtual CPUs to allocate to the vApp
+* `initscript` (Optional; **Deprecated**) A script to be run only on initial boot
+* `network_name` - (Optional; **Deprecated**) Name of the network this vApp should join
+* `storage_profile` - (Optional)
+* `ip` - (Optional; **Deprecated**) The IP to assign to this vApp. Must be an IP address or
   one of dhcp, allocated or none. If given the address must be within the
   `static_ip_pool` set for the network. If left blank, and the network has
   `dhcp_pool` set with at least one available IP then this will be set with
   DHCP.
 * `metadata` - (Optional) Key value map of metadata to assign to this vApp. Key and value can be any string. (Since *v2.2+* metadata is added directly to vApp instead of first VM in vApp)
-* `ovf` - (Optional) Key value map of ovf parameters to assign to VM product section
+* `ovf` - (Optional; **Deprecated**) Key value map of ovf parameters to assign to VM product section
 * `power_on` - (Optional) A boolean value stating if this vApp should be powered on. Default is `true`
 * `org` - (Optional; *v2.0+*) The name of organization to use, optional if defined at provider level. Useful when connected as sysadmin working across different organisations
 * `vdc` - (Optional; *v2.0+*) The name of VDC to use, optional if defined at provider level
-* `accept_all_eulas` - (Optional; *v2.0+*) Automatically accept EULA if OVA has it. Default is `true`
+* `accept_all_eulas` - (Optional; *v2.0+*; **Deprecated**) Automatically accept EULA if OVA has it. Default is `true`
 * `guest_properties` - (Optional; *v2.5+*) Key value map of vApp guest properties **Note** `ovf` attribute
 sets guest properties on the first VM using a legacy ability of this resource to spawn 1 VM. Please
 use resources `vcd_vapp_vm` to provision VMs.
+* `href` - (Computed) The vApp Hyper Reference
+* `status` - (Computed) The vApp status as a numeric code
+* `status_text` - (Computed) The vApp status as text.
+
+## Importing
+
+Supported in provider *v2.5+*
+
+~> **Note:** The current implementation of Terraform import can only import resources into the state. It does not generate
+configuration. [More information.][docs-import]
+
+An existing vApp can be [imported][docs-import] into this resource via supplying its path.
+The path for this resource is made of org-name.vdc-name.vapp-name.
+For example, using this structure, representing a vapp that was **not** created using Terraform:
+
+```hcl
+resource "vcd_vapp" "tf-vapp" {
+  name              = "my-vapp"
+  org               = "my-org"
+  vdc               = "my-vdc"
+}
+```
+
+You can import such vapp into terraform state using this command
+
+```
+terraform import vcd_vapp.tf-vapp my-org.my-vdc.my-vapp
```

Comment:
For clarity, I propose adding "-name" for these references.

Reply:
The example is using the names in the sample script. If I add `-name`, the example won't work.